.github/workflows/ci.yml

```diff
@@ -33,9 +36,20 @@ jobs:
         run: |
           crystal spec
           crystal spec --release --no-debug
+        continue-on-error: ${{ matrix.allowed-failure }}
 
       - name: Check formatting
         run: crystal tool format --check
 
-      - name: Run ameba linter
-        run: bin/ameba
+  check_ameba:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Crystal
+        uses: crystal-lang/install-crystal@v1
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: Crystal Ameba Linter
+        id: crystal-ameba
+        uses: crystal-ameba/github-action@v0.7.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The action seems to be broken (ref https://github.com/kemalcr/kemal/actions/runs/5443532334/jobs/9900164974?pr=657). So better exclude that from this PR. It's not necesary for Windows support.